public/src/components/PostItem/CommentsSection.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import CommentComponent from './CommentComponent';
+import methods from '../../utils/methods';
+
+const { sortComments } = methods;
+
+export default class CommentsSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.commentToShowStep = 20;
+        this.getComments = this.getComments.bind(this);
+        this.showMoreComments = this.showMoreComments.bind(this);
+        this.state = {
+            commentsToShow: this.commentToShowStep,
+        };
+    }
+
+    getComments() {
+        const { commentsToShow } = this.state;
+        const { comments } = this.props;
+        let sortedComments = sortComments([...comments]);
```

Comment:
Do you need `let` here? It seems like you can just continue chaining

```js
const sortedComments = sortComments(comments).filter(...).slice(...)
```

Reply:
fixed